tests/v2/validation/rbac/secrets/rbac_secrets_test.go

```diff
@@ -0,0 +1,283 @@
+//go:build (validation || infra.any || cluster.any || extended) && !sanity && !stress
+
+package secrets
+
+import (
+	"strings"
+	"testing"
+
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/shepherd/clients/rancher"
+	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+	"github.com/rancher/shepherd/extensions/clusters"
+	"github.com/rancher/shepherd/extensions/kubeapi/namespaces"
+	rbac "github.com/rancher/shepherd/extensions/rbac"
+	"github.com/rancher/shepherd/extensions/secrets"
+	"github.com/rancher/shepherd/extensions/users"
+	namegen "github.com/rancher/shepherd/pkg/namegenerator"
+	"github.com/rancher/shepherd/pkg/session"
+	"github.com/rancher/shepherd/pkg/wrangler"
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type RbacSecretTestSuite struct {
```

Comment:
We are declaring too many types in this struct, usually its not a good practice to have them all defined in the global struct.. do we need downstreamContext, standardUser, standardUserClient,  namespaceName declared here? It is ok to have them created in each test rather.